tests/integration/tables/apps.cpp

```diff
@@ -14,46 +14,38 @@
 namespace osquery {
 namespace table_tests {
 
-class apps : public testing::Test {
+class AppsTest : public testing::Test {
  protected:
   void SetUp() override {
     setUpEnvironment();
   }
 };
 
-TEST_F(apps, test_sanity) {
-  // 1. Query data
+TEST_F(AppsTest, test_sanity) {
   auto const data = execute_query("select * from apps");
-  // 2. Check size before validation
-  // ASSERT_GE(data.size(), 0ul);
-  // ASSERT_EQ(data.size(), 1ul);
-  // ASSERT_EQ(data.size(), 0ul);
-  // 3. Build validation map
-  // See helper.h for avaialbe flags
-  // Or use custom DataCheck object
-  // ValidationMap row_map = {
-  //      {"name", NormalType}
-  //      {"path", NormalType}
-  //      {"bundle_executable", NormalType}
-  //      {"bundle_identifier", NormalType}
-  //      {"bundle_name", NormalType}
-  //      {"bundle_short_version", NormalType}
-  //      {"bundle_version", NormalType}
-  //      {"bundle_package_type", NormalType}
-  //      {"environment", NormalType}
-  //      {"element", NormalType}
-  //      {"compiler", NormalType}
-  //      {"development_region", NormalType}
-  //      {"display_name", NormalType}
-  //      {"info_string", NormalType}
-  //      {"minimum_system_version", NormalType}
-  //      {"category", NormalType}
-  //      {"applescript_enabled", NormalType}
-  //      {"copyright", NormalType}
-  //      {"last_opened_time", NormalType}
-  //}
-  // 4. Perform validation
-  // validate_rows(data, row_map);
+  ASSERT_GE(data.size(), 1ul);
+  ValidationMap row_map = {
+      {"name", NonEmptyString},
+      {"path", DirectoryOnDisk},
+      {"bundle_executable", NormalType},
+      {"bundle_identifier", NormalType},
+      {"bundle_name", NormalType},
+      {"bundle_short_version", NormalType},
+      {"bundle_version", NormalType},
+      {"bundle_package_type", NormalType},
+      {"environment", NormalType},
+      {"element", NormalType},
+      {"compiler", NormalType},
+      {"development_region", NormalType},
+      {"display_name", NormalType},
+      {"info_string", NormalType},
+      {"minimum_system_version", NormalType},
+      {"category", NormalType},
+      {"applescript_enabled", NormalType},
```

Comment:
I think we want to address this. Right now it seems the values are either empty or `1`.

Reply:
also have different values for those fields
- `minimum_system_version`
  - `10.7`
  - `10.14`
- `info_string`
  - `Eclipse 4.13 for Mac OS X, Copyright IBM Corp. and others 2002, 2019. All rights reserved.`
  - 3.7.4, © 2001-2019 Python Software Foundation
  - ...